### DISCUSSION TITLE
Himmelblau logs

### DISCUSSION THREAD

**Discussion by securitymonster** [himmelblau-idm/himmelblau#211] [General]
Hi, I have compiled and configured Himmelblau on Ubuntu. The service is running, I've configured my tenants domain and group-ID, setup nsswitch.conf. However when I try getent passwd <username> I get no response, it returns to the prompt. 

I used strace on getent, it returns {"NssAccount":null}. I tried a tcpdump on the IP of the domain name and also on 443, but no packets get send out.

Are there any Himmelblau logs I can view?

**Comment by dmulder** [comment]
The logs all go to the syslog. I recently added an option to the himmelblau.conf to enable debug, but it doesn't seem to be working (I need to fix that). What does work is you can pass the parameter `-d` to the himmelblaud daemon, and it will send debug to the syslog. (`journalctl -f himmelblaud`).
I did a debugging session with @run-stop on Ubuntu the other day, and we encountered #205. Make sure the domain name you set in the himmelblau.conf matches the domain in your upn (the domain part of your username).

**Comment by dmulder** [comment]
Oh, and FYI, you don't need to build Himmelblau yourself on Ubuntu. There are packages: https://github.com/himmelblau-idm/himmelblau/releases/tag/0.6.0

**Comment by dmulder** [comment]
> Hi, I have compiled and configured Himmelblau on Ubuntu. The service is running, I've configured my tenants domain and group-ID, setup nsswitch.conf. However when I try getent passwd I get no response, it returns to the prompt.
> 
> I used strace on getent, it returns {"NssAccount":null}. I tried a tcpdump on the IP of the domain name and also on 443, but no packets get send out.

Did you get this resolved? I'm happy to have a virtual meeting with you to debug the problem.

**Comment by iLikeToCode** [comment]
Closing as discussion has been stale for over 10 months.
